### DISCUSSION TITLE
Project timeline

### DISCUSSION THREAD

**Discussion by fliebau** [intlify/vue-i18n#930] [General]
Hey there :)

Just a general question about the project timeline. I've seen that version 9.2 is since a while in beta and would like to ask if the team has any date in mind to release it? Thank you for your great work and looking forward to hear from you! :)

Cheers, 
Falco

**Comment by kazupon** [comment]
I'm focusing nuxt/i18n supporting with nuxt community team now
https://github.com/nuxt-community/i18n-module/discussions/1287

In there, We are using vue-i18n version 9.2 beta, and a new package I created called vue-i18n-routing.
I will officially release vue-i18n verions 9.2 with vue-i18n-routing, when nuxt/i18n is almost ready to support nuxt 3 / nuxt bridge.
